cmd/kubeadm/app/apis/kubeadm/v1alpha3/defaults_unix.go

```diff
@@ -23,6 +23,6 @@ const (
 	DefaultCACertPath = "/etc/kubernetes/pki/ca.crt"
 	// DefaultSocketUrlScheme defines default socket url prefix
 	DefaultUrlScheme = "unix"
-	// DefaultCRISocket defines the default cri socket
-	DefaultCRISocket = "/var/run/dockershim.sock"
+	// DefaultDockerCRISocket defines the default Docker CRI socket
```

Comment:
Now that v1beta1 merged those changes should be implemented there.
With regards to implement changes on v1alpha3 as well, I think that we are not allowed to alter the behaviour of an existing API versions, but I'm open to discuss if this case (change of a default for improving UX) is an acceptable exception